lib/childProcess.js

```diff
@@ -1,24 +1,24 @@
-const crossSpawn = require('cross-spawn');
-
-const ignorePipeErrors = require('./ignorePipeErrors');
+import crossSpawn from 'cross-spawn';
 
+import ignorePipeErrors from './ignorePipeErrors';
 
 const ASCII_CTRL_C = 3;
 const IS_WINDOWS = process.platform === 'win32';
 const TERM_FIRST_CHECK_TIMEOUT_MS = 1;
 const TERM_DEFAULT_TIMEOUT_MS = 1000;
 const TERM_DEFAULT_RETRY_MS = 300;
 
-
 // Signals the child process to forcefully terminate
-function signalKill(childProcess, callback) {
+export const signalKill = (childProcess, callback) => {
```

Comment:
What's the reason for changing from a function to closure here? Seems inconsistent with some of the code in prior files such as `function formatLocation(apiDescriptionLocation, annotationLocation) {` above.